autopush_rs/src/util/autojson.rs

```diff
@@ -0,0 +1,287 @@
+// {{{ Crate docs
```

Comment:
will all of this file go away w/ the slog enhancements?

Reply:
Yup.